### PR TITLE
GUI-2199: Better handling of alarm name in Copy Alarm dialog

### PR DESCRIPTION
**Pull request by kamalgill**
Minor tweaks for https://eucalyptus.atlassian.net/browse/GUI-2199

### CODE REVIEW

File: eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js [hunk 1]
```diff
@@ -137,7 +134,11 @@ angular.module('CreateAlarmModal', [
                     AlarmService.getAlarm(attrs.alarmName)
                         .then(function (res) {
                             var alarm = res.alarm;
+                            alarm.name = '';
                             vm.initializeForCopy(alarm, attrs);
+                            var nameField = angular.element('[name="createAlarmForm"]').find('[name="name"]');
+                            nameField.focus();
+                            $scope.createAlarmForm.name.$touched = false;
```
Comment (coyote240) [inline eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js]: As $scope.createAlarmForm.name gets you a reference to the form field, the use of angular.element above may not be necessary.

Comment (kamalgill) [inline eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js]: @coyote240 : Changing lines 139-140 to `$scope.createAlarmForm.name.focus()` doesn't work, logging the following in the JS console...

```
Error: $scope.createAlarmForm.name.focus is not a function. (In '$scope.createAlarmForm.name.focus()', '$scope.createAlarmForm.name.focus' is undefined)
```

Comment (coyote240) [inline eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js]: $scope.createAlarmForm.name is not the element, but it contains a reference to the element.

Comment (kamalgill) [inline eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js]: $scope.createAlarmForm.name doesn't appear to contain a reference to the element, and angular.element($scope.createAlarmForm.name) doesn't seem to work either, so will leave the code as-is for now.

File: eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js [hunk 2]
```diff
@@ -49,11 +46,15 @@ angular.module('CreateAlarmModal', [
 
             $scope.$watchCollection('alarm', function (newVal) {
                 if(newVal.metric && $scope.createAlarmForm.name.$untouched) {
-                    $scope.alarm.name = $scope.alarmName();
+                    $scope.alarm.name = $scope.setAlarmName();
```
Comment (dkavanagh) [inline eucaconsole/static/js/widgets/create-alarm-modal/create-alarm.js]: A method that returns a value probably shouldn't be called "set..."

**Pull request merged**